web/server/api/endpoints/lineage.py

```diff
@@ -78,20 +75,17 @@ async def column_lineage(
     for i, node in enumerate(node.walk()):
         if i > 0:
             node_name = _get_table(node) or node.name
-            node_name = (
-                context.snapshots[node_name].name if node_name in context.snapshots else node_name
-            )
+            node_name = normalize_model_name(node_name, None)
```

Comment:
this shouldn't be necessary

Reply:
without `normalize_model_name` node (table) name coming without quotes 

![Screenshot 2023-12-28 at 9 44 08 AM](https://github.com/TobikoData/sqlmesh/assets/5644753/afe3b47f-8fcd-4ee6-ab7c-82f9b9df0bf2)

Reply:
the issue is you're calling _get_table which is incorrect. instead it should be 

exp.table_name(node, identify=True)

Reply:
so add identify=True into _get_table helper